### PR TITLE
Reji/update samples

### PR DESCRIPTION
**Pull request by rejik2**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,77 +1,83 @@
...
     });
 }
 
-onexecute = function(objectName, methodName, parameters, properties) {
-    switch (objectName)
-    {
-        case "com.sample.odata.people": onexecuteGet(methodName, parameters, properties); break;
+onexecute = async function (objectName, methodName, parameters, properties): Promise<void> {
+    switch (objectName) {
+        case "people": onexecuteGet(methodName, parameters, properties); break;
```
Comment (ryanelfmank2) [inline src/index.ts]: doesn't this need to be awaited?

File: src/index.ts [hunk 2]
```diff
@@ -1,77 +1,83 @@
...
     }
 }
 
-function onexecuteGet(methodName: string, parameters: SingleRecord, properties: SingleRecord) {
-    switch (methodName)
-    {
-        case "com.sample.odata.people.get": onexecuteGetPeople(parameters, properties); break;
+async function onexecuteGet(methodName: string, parameters: SingleRecord, properties: SingleRecord): Promise<void> {
+    switch (methodName) {
+        case "get": onexecuteGetPeople(parameters, properties); break;
```
Comment (ryanelfmank2) [inline src/index.ts]: await

File: src/index.ts (hunk 1 above)
Comment (michta42) [reply src/index.ts]: And can you remove the dots everywhere? In this example, com.sample.odata.people should just be people.

Comment (rejik2) [reply src/index.ts]: @michta42 It is removed. The Red color is old code. The green color is changed code

File: src/index.ts (hunk 2 above)
Comment (rejik2) [reply src/index.ts]: done

File: src/index.ts (hunk 1 above)
Comment (rejik2) [reply src/index.ts]: @ryanelfmank2 added await

**Review by ryanelfmank2 — APPROVED** [review]

**Review by jcdickinson — APPROVED** [review]

**Pull request merged**